user/jwt.md

```diff
@@ -0,0 +1,156 @@
+---
+title: JWT addon
+layout: en
+permalink: /user/jwt/
+---
+
+Integration between Travis-CI and third-party services like Sauce Labs relies
+on [encrypted variables](http://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables)
+which works well for trusted branches and committers. For security reasons,
+encrypted variables are not exposed to untrusted pull requests, so third-party
+integrations do not work for pull requests.
```

Comment:
This paragraph reads like a non sequitur, even though it isn't really.  Could you have a go at rewording it?